### DISCUSSION TITLE
Performance of Microsoft's OpenJDK Vs Other JDKs

### DISCUSSION THREAD

**Discussion by Brettb-gpu** [microsoft/openjdk#92] [Q&A] [answered]
Are there any real statistics how well this build of OpenJDK compares with AdoptOpenJDK or JDKs that use different JVMs in general such as GraalVm?

**Comment by karianna** [comment, marked as answer]
We haven't completed or published our performance benchmarks. This is definitely on our short-term roadmap, we believe in transparency on this topic but unfortunately, I can't give a concrete timeline on the delivery of this just yet.

That said you can likely assume that we have the same performance profile as AdoptOpenJDK as we build from the same source files and use very similar build toolchains.

When we do our performance benchmarking we'll explore if it is possible (it may not be for various reasons) to do a Apples to Apples comparison with other JVM implementations such as Azul's Zing, OpenJ9, or GraalVM.  Thanks for the feedback there!
